docs/multitenant.md

```diff
@@ -0,0 +1,36 @@
+# Multitenant
+
+This bundle includes an multitenant management.
+You can enable the multitenant mode following this instructions:
+
+## Configuration
+Add the FCQN in `tenant_class` line in `config/packages/lle_hermes.yaml`:
+
+```yaml
+# config/packages/lle_hermes.yaml
+lle_hermes:
+    tenant_class: App\Entity\Establishment
+```
+
+In your user entity implements the `MultiTenantInterface` with the method `getTenantId()`:
```

Comment:
implement sans s et mets une virgule avant.

In your user entity, implement the ...